src/utils.js

```diff
@@ -57,7 +58,7 @@ function noop() {}
 export function subtractMonth({ calendars, offset, minDate }) {
   if (offset > 1 && minDate) {
     let { firstDayOfMonth } = calendars[0];
-    let diffInMonths = monthDiff(minDate, firstDayOfMonth);
+    let diffInMonths = differenceInCalendarMonths(firstDayOfMonth, minDate);
```

Comment:
While `monthDiff` had the signature (earlierDate, laterDate), [differenceInCalendarMonths](https://date-fns.org/v1.29.0/docs/differenceInCalendarMonths#arguments) has the opposite.